.travis.yml

```diff
@@ -13,3 +12,10 @@ before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
   - sleep 3 # give xvfb some time to start
+
+script:
+  - npm run test:$TEST_MODE
```

Comment:
Can't you just call `npm run test`? Aren't the ENV vars below available? Or did you add them for convenience?